cf_units/__init__.py

```diff
@@ -187,15 +185,8 @@ def suppress_errors():
     try:
         _ud_system = _ud.read_xml()
     except _ud.UdunitsError:
-        _alt_xml_path = config.get_option(
-            "System",
-            "udunits2_xml_path",
-            default=os.path.join(
-                sys.prefix, "share", "udunits", "udunits2.xml"
-            ),
-        )
         try:
-            _ud_system = _ud.read_xml(_alt_xml_path.encode())
+            _ud_system = _ud.read_xml(config.get_xml_path())
```

Comment:
`codecov` `patch` was complaining about this line change, which is pretty impossible to explicitly test... so I turned `patch` `off`, which is a reasonable stance to take.

The behaviour of `cf-units` here is pretty fundamental to the `wheels` installation, and this line is being exercised successfully during a wheels install, otherwise no tests would run (nor would `cf-units` import) from a wheels install. And we know that GHA testing of a wheels installation is good, and normal CI testing is good, so I'm okay with this.